packages/terra-consumer-nav/src/components/nav-help/NavHelpContent.jsx

```diff
@@ -67,11 +68,12 @@ class NavHelpContent extends React.Component {
               </p>))
           }
           </Toggler>
-        </Button>);
+        </div>);
       } else {
+        const helpItemStyle = cx('help-item', index > 0 ? 'help-item-border' : '');
```

Comment:
same class def comment

Reply:
maybe move the style def outside the if/else if it's always going to be applied in both scenarios